static/app/views/performance/utils.tsx

```diff
@@ -581,6 +582,46 @@ export function getIntervalLine(
       }),
       data: [],
     });
+
+    additionalLineSeries.push({
+      seriesName: 'Baseline Axis Line',
+      type: 'line',
+      markLine:
+        MarkLine({
+          silent: true,
+          label: {
+            show: false,
+          },
+          lineStyle: {color: theme.green400, type: 'solid', width: 4},
+          data: [
+            [
+              {xAxis: 'min', y: 177},
+              {xAxis: breakpoint, y: 177},
+            ],
+          ],
+        }) ?? {},
+      data: [],
+    });
+
+    additionalLineSeries.push({
+      seriesName: 'Regression Axis Line',
+      type: 'line',
+      markLine:
+        MarkLine({
+          silent: true,
+          label: {
+            show: false,
+          },
+          lineStyle: {color: theme.red300, type: 'solid', width: 4},
+          data: [
+            [
+              {xAxis: breakpoint, y: 177},
+              {xAxis: 'max', y: 177},
```

Comment:
Yeah, I didn't really want to but I didn't have a reliable way of finding the location of the x-axis. It's just some margin away from the edge of the SVG based on a height of 200px

I'll a comment for now and calculate it with constants since this is a fixed height chart. The offset is configurable but we rarely do that so this should work for our use case right now